### DISCUSSION TITLE
[REQUEST] Komga/Mylar3 and Podcasts

### DISCUSSION THREAD

**Discussion by spocoeric** [tteck/Proxmox#3152] [General]
I would love to have scripts for Komga comic book server and Mylar3 automated comic book downloader.

Would also welcome a podcast grabber, either Podgrab or PodFetch.

**Comment by niicholai** [comment]
Would absolutely love to see a Komga LXC script, I prefer it over Kavita as it seems to work better in my experience and is very smooth integrating with apps like TachiJ2K.
